indexer-app/src/cmr/indexer/data/concepts/service.clj

```diff
@@ -58,13 +58,15 @@
        :user-id user-id
        :revision-date revision-date
        :metadata-format (name (mt/format-key format))
-       :associations-gzip-b64 (util/string->gzip-base64
-                               (pr-str
-                                (util/remove-map-keys
-                                 empty?
-                                 (assoc-util/assoc-list->assoc-struct
-                                  (concat service-associations generic-associations)
-                                   concept-id))))})))
+       :associations-gzip-b64 (when (or (seq service-associations)
+                                        (seq generic-associations))
+                                (util/string->gzip-base64
+                                 (pr-str
+                                  (util/remove-map-keys
+                                   empty?
+                                   (assoc-util/assoc-list->assoc-struct
+                                    (concat service-associations generic-associations)
+                                     concept-id)))))})))
```

Comment:
Would recommend simplification with thrush, and concat the associations higher up into a single list.
```suggestion
       :associations-gzip-b64 (when (seq any-assocs)
                                        (->> concept-id 
                                                 (assoc-util/assoc-list->assoc-struct any-assocs)
                                                 (util/remove-map-keys empty?)
                                                 pr-str
                                                 util/string->gzip-base64)
```